package.json

```diff
@@ -19,8 +19,8 @@
   },
   "scripts": {
     "watch": "tsc -w",
-    "build": "rm -rf dist; tsc -d",
-    "prepublish": "npm run build; npm test",
+    "build": "rimraf dist; tsc -d",
```

Comment:
@joelgriffith hm, it is here, but  looks like some merge conflict.. There are 2 `devDependencies ` now